### PR TITLE
adding SG servers and adding comments

### PR DESCRIPTION
**Pull request by vincentserpoul**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -19,6 +13,13 @@ import (
 	"sync"
 	"sync/atomic"
 	"time"
+
```
Comment (Philipp15b) [inline client.go]: The imports should always be sorted with `go fmt`, not manually. Could you please run `go fmt` and rebase?

Comment (vincentserpoul) [inline client.go]: Totally agree, imports should not be sorted manually and use go fmt.
I'm using go imports tool for google codebase, along with other go tools:

go get -u golang.org/x/tools/cmd/goimports
go get -u golang.org/x/tools/cmd/vet
go get -u golang.org/x/tools/cmd/oracle
go get -u golang.org/x/tools/cmd/godoc

Tell me if you disagree, I think it brings normalization to another level and it's provided by golang teams themselves.

But it's your repo, I will stick to your will :)

Comment (vincentserpoul) [inline client.go]: any comment on this?

Comment (Philipp15b) [inline client.go]: Sorry for letting you wait so long. Deciding about using `goimports` or not rather feels like tabs vs spaces, there's no definitive right choice. I'll merge this and use `goimports` in the future :) Thank you for your contribution!

Comment (vincentserpoul) [inline client.go]: agreed, thanks @Philipp15b

**Pull request merged**